components/Grid/grid.css

```diff
@@ -0,0 +1,19 @@
+.grid{
+    display : grid;
+}
+
+.grid-ratio-50-50{
+    grid-template-columns: repeat(2, 1fr);
+}
+
+.grid-ratio-30-70{
+    grid-template-columns : 1fr 2fr;
+}
+
+.grid-ratio-70-30{
+    grid-template-columns: 2fr 1fr;
+}
+
+.grid-three-item{
+    grid-template-columns : repeat(3 , 1fr);
+}
```

Comment:
ideally, you should have written 7fr 3fr for 70-30 grid and 3fr 7fr for 30-70 grid. class naming is simple and elegant.